components/backup/src/writer.rs

```diff
@@ -190,6 +199,76 @@ impl BackupWriter {
     }
 }
 
+/// A writer writes Raw kv into SST files.
+pub struct BackupRawKVWriter {
+    name: String,
+    cf: String,
+    write: Writer,
+    limiter: Option<Arc<RocksIOLimiter>>,
+}
+
+impl BackupRawKVWriter {
+    /// Create a new BackupRawKVWriter.
+    pub fn new(
+        db: Arc<DB>,
+        name: &str,
+        cf: &str,
+        limiter: Option<Arc<RocksIOLimiter>>,
+    ) -> Result<BackupRawKVWriter> {
+        let write = RocksSstWriterBuilder::new()
```

Comment:
The name of the fields `default` and `write` in `BackupWriter` means different CFs. Here I don't think `write` is a proper name. Consider name it `writer`.

Reply:
change write to writer